kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -265,10 +293,17 @@ public void writeSinkRecords(Collection<SinkRecord> records) {
 
     for (SinkRecord record : records) {
       if (record.value() != null || config.getBoolean(BigQuerySinkConfig.DELETE_ENABLED_CONFIG)) {
-        PartitionedTableId table = getRecordTable(record);
+        PartitionedTableId table = useStorageApi ? getStorageApiRecordTable(record.topic()) : getRecordTable(record);
```

Comment:
Just a suggestion - I think we should maintain consistency incase of methods fetching the same values. Either we should sent the `record` or `record.topic()` in both the methods.

Reply:
Though they are returning the same type of response, these are 2 different methods. And the built response has same type but content can be different. `getStorageApiRecordTable` does not require the entire record. If we send the SinkRecord we will get the topic only there.

Reply:
Sure, this makes sense.